### DISCUSSION TITLE
How to Create a Component Library from Quasar Component ?

### DISCUSSION THREAD

**Discussion by fmgon0** [quasarframework/quasar#14289] [General - Components / Directives / etc]
Is there any tips how to create a shared component library package but with Quasar Component as based components ?
Cause we have a design system, and currently we are using monorepo. And still don't have a singel package UI to be shared.
Cause in every project, we have to rewrite our component style from another project, and it make us not productive.

Any tips from you guys how to make this right ?

**Comment by corepay** [comment]
This might help you setup and register global components.  https://www.youtube.com/watch?v=SiQUlLSa4sA
